packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -522,6 +528,12 @@ export class TdsDropdown {
                   }}
                 />
               </div>
+              <tds-icon
+                onClick={this.handleFilterReset}
+                class={`${this.open && this.inputElement.value !== '' ? '' : 'hide'}`}
+                name="cross"
+                size="16px"
+              ></tds-icon>
```

Comment:
I'd consider making this reachable by keyboard, wrapping it in a button would create semantically incorrect HTML I believe. But should be able to move it outside of the input element in the DOM and then position it correctly using CSS.

Reply:
Ok, will have a look at it when I get the time.